src/org/zaproxy/zap/extension/requester/RequesterOptionsPanel.java

```diff
@@ -0,0 +1,107 @@
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ */
+package org.zaproxy.zap.extension.requester;
+
+import java.awt.CardLayout;
+import java.awt.GridBagLayout;
+import java.awt.Insets;
+
+import javax.swing.JCheckBox;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.border.EmptyBorder;
+
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.model.OptionsParam;
+import org.parosproxy.paros.view.AbstractParamPanel;
+import org.zaproxy.zap.view.LayoutHelper;
+
+/**
+ * The GUI breakpoints options panel.
+ * <p>
+ * It allows to change the following breakpoints options:
+ * <ul>
+ * <li>Confirm drop message - asks for confirmation when a trapped message is dropped.</li>
+ * </ul>
+ * </p>
+ * 
+ * @see org.zaproxy.zap.extension.brk.BreakPanelToolbarFactory#getBtnDrop()
+ */
+public class RequesterOptionsPanel extends AbstractParamPanel {
+
+    private static final long serialVersionUID = 1L;
+
+    private JCheckBox checkBoxAutoFocus = null;    
+
+    public RequesterOptionsPanel() {
+        super();
+        setName(Constant.messages.getString("requester.optionspanel.name"));
+
+        this.setLayout(new CardLayout());
+
+        JPanel panel = new JPanel(new GridBagLayout());
+        panel.setBorder(new EmptyBorder(2, 2, 2, 2));
+
+        panel.add(getCheckBoxAutoFocus(), LayoutHelper.getGBC(0, 0, 2, 1.0, new Insets(2, 2, 2, 2)));       
+        
+        
+		panel.add(new JLabel(), LayoutHelper.getGBC(0, 10, 1, 0.5D, 1.0D));	// Spacer
+
+        add(panel);
+    }    
+    
+    private JCheckBox getCheckBoxAutoFocus() {
+    	if (checkBoxAutoFocus == null) {
+    		checkBoxAutoFocus = new JCheckBox(
+    				Constant.messages.getString("requester.optionspanel.option.autoFocus.label"));
+    	}
+    	return checkBoxAutoFocus;
+    }
+
+
+    @Override
+    public void initParam(Object obj) {
+        final OptionsParam options = (OptionsParam) obj;
+        final RequesterParam param = options.getParamSet(RequesterParam.class);
+
+        
+        // Note param.alwaysOnTop will be null if the user hasnt specified a preference yet
+        getCheckBoxAutoFocus().setSelected(param.getAutoFocus());
+        
+    }
+
+    @Override
+    public void validateParam(Object obj) throws Exception {
+    }
+
+    @Override
+    public void saveParam(Object obj) throws Exception {
+        final OptionsParam options = (OptionsParam) obj;
+        final RequesterParam param = options.getParamSet(RequesterParam.class);
+
+        param.setAutoFocus(getCheckBoxAutoFocus().isSelected());        
+    }
+
+    @Override
+    public String getHelpIndex() {
+        return "ui.dialogs.options.requester";
```

Comment:
This should return null, or help should be provided - at the moment clicking on the help icon causes an exception.